storage/mdcb_storage.go

```diff
@@ -8,12 +8,19 @@ import (
 )
 
 type MdcbStorage struct {
-	local                 Handler
-	rpc                   Handler
-	logger                *logrus.Entry
-	CallbackonPullfromRPC *func(key string, val string) error
+	local                            Handler
+	rpc                              Handler
+	logger                           *logrus.Entry
+	CallbackOnPullCertificateFromRPC *func(key string, val string) error
```

Comment:
Also, is there any advantage of keeping this type as a func pointer?

Reply:
Is it still possible for this pointer to be nil?

Reply:
changed to not pointer, unsure why we had it like this when even not being a pointer it can be nil